sig-federation/ONCALL.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+We have an oncall rotation in the SIG. The role description is as follows:
+
+* Ensure that the testgrid (https://k8s-testgrid.appspot.com/sig-federation) is green. This person will be the point of contact if testgrid turns red. Will identify the problem and fix it (most common scenarios: find culprit PR and revert it or free quota by deleting leaked resources).
+Will also report most common failure scenarios and suggest improvements. Its up to the sig or individuals to prioritize and take up those tasks.
+
+Oncall playbook: https://github.com/kubernetes/community/blob/master/contributors/devel/on-call-federation-build-cop.md
+
+# Joining the rotation
+
+Add your name at the end of the current rotation schedule.
+
+# Oncall schedule
+
+```
+25 September - 1 October: Madhu (https://github.com/madhusudancs)
+2 October - 8 October: Shashidhara (https://github.com/shashidharatd)
+9 October - 15 October: Christian (https://github.com/csbell)
+16 October - 22 October: Nikhil Jindal (https://github.com/nikhiljindal)
+23 October - 29 October: Irfan (https://github.com/irfanurrehman)
+30 October - 5 November: Maru (https://github.com/marun)
+6 November - 12 November: Jonathan (https://github.com/perotinus)
+(Madhu to be removed from next rotation)
+
+(Adding Irfan)
```

Comment:
Can we get rid of (or gray out) the history?  I assume it's here so that we have a record and can extend the rotation fairly?  It's a bit distracting for people who are only looking for who's oncall today.

Reply:
Have put it in a different Past cycles section.